src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -179,6 +183,107 @@ class Cas2ApplicationTest : IntegrationTestBase() {
   @Nested
   inner class GetToIndex {
 
+    @Test
+    fun `return unexpired applications when applications GET is requested`() {
+      val unexpiredSubset = setOf(
```

Comment:
does this cover applications without a status update as well?

Reply:
No it didn't, good point.  I've added another couple of applications which don't have status updates.